jdatetime/__init__.py

```diff
@@ -39,6 +39,7 @@
 
 if sys.version_info[0] >= 3:  # py3
     _int_types = (int,)
+    basestring = str
```

Comment:
perhaps it's better to make this variable private? (`basestring` -> `_basestring`)